config/single-catalyst.json

```diff
@@ -0,0 +1 @@
+{"account":{"test":{"password":"098f6bcd4621d373cade4e832627b4f6"}},"autopushes":null,"bandwidth":{"exceptions":[],"limit":0},"config":{"accesslog":"LOG","controller":{"interface":null,"port":null,"username":null},"debug":null,"defaultStream":null,"limits":null,"location":{"lat":0.0000000000,"lon":0.0000000000,"name":""},"prometheus":"koekjes","protocols":[{"connector":"AAC"},{"connector":"CMAF"},{"connector":"DTSC"},{"connector":"EBML"},{"connector":"FLV"},{"connector":"H264"},{"connector":"HDS"},{"connector":"HLS"},{"connector":"HTTP"},{"connector":"HTTPTS"},{"connector":"JSON"},{"connector":"MP3"},{"connector":"MP4"},{"connector":"OGG"},{"connector":"RTMP"},{"connector":"RTSP"},{"connector":"SDP"},{"connector":"SRT"},{"connector":"TSSRT"},{"connector":"WAV"},{"connector":"WebRTC"},{"connector":"livepeer-catalyst-node","redirect-prefixes":"stream","rpc-addr":"0.0.0.0:7373"}],"serverid":null,"sessionInputMode":"14","sessionOutputMode":"14","sessionStreamInfoMode":"1","sessionUnspecifiedMode":"0","sessionViewerMode":"14","sidMode":"0","triggers":{"USER_NEW":[{"default":"true","handler":"http://127.0.0.1:8090/triggers","streams":[],"sync":true}]},"trustedproxy":[]},"push_settings":{"maxspeed":null,"wait":null},"streams":{"stream":{"name":"stream","processes":[],"realtime":false,"source":"balance:http://localhost:8042/?fallback=push://","stop_sessions":false}},"ui_settings":null}
```

Comment:
is the file needed?

Reply:
This file, as other config files are mainly used for local setup and exposed as examples. What bothers me is that MistController keep rewriting in a single line. Will update it to the adequate formatting.

Reply:
oh, ok. you can use `make dev` which creates a copy to `~/.config/livepeer/catalyst.json` and uses that for starting up `MistController`.